src/main/java/net/openhft/chronicle/threads/Threads.java

```diff
@@ -128,9 +128,8 @@ public static void shutdown(@NotNull ExecutorService service, boolean daemon) {
     public static void shutdown(@NotNull ExecutorService service) {
 
         service.shutdown();
-        // without these 2 here, some threads that were in a LockSupport.parkNanos were taking a long time to shut down
+        // without this here, some threads that were in a LockSupport.parkNanos were taking a long time to shut down
         Threads.unpark(service);
-        Threads.interrupt(service);
```

Comment:
Note that `service.shutdownNow()` is called on line 136 below after the timeout

https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html#shutdownNow--

Reply:
I added the call to `Threads.interrupt` because BlockingEventLoop was taking a long time to shut down - we were getting warnings saying that the thread had not terminated, and the stack trace showed it was stuck in the Jvm.pause below. 
```
   void doPause(long delayNs) {
        ...
        if (!Thread.currentThread().isInterrupted()) {
            if (delayNs >= 1_000_000)
                Jvm.pause(delayNs / 1000000);
            else
                LockSupport.parkNanos(delayNs);
        }
        ...
    }
```
@peter-lawrey I believe you added in the if that now calls Jvm.pause. Previously it just called LockSupport.parkNanos.

Reply:
Shouldn't BlockingEventLoop.close be the one doing the interrupting in that scenario?

Reply:
I reckon we also need https://github.com/OpenHFT/Chronicle-Threads/pull/86